chapter2/exercise2.js

```diff
@@ -0,0 +1,29 @@
+(function() {
+
+    displayDivisibilityNumbers = function() {
+        var numberDivideByThree = false, numberDivideByFive = false, number = 1, result = "";
```

Comment:
prosze nie uzywal single var pattern. jest to najgorszy z mozliwych patternow dla moich oczu :dagger:

Reply:
Okej, zatem rozumiem, że wolisz kiedy każda zmienna jest osobno deklarowana? Nie sądzisz, ze tego typu deklarowanie każdej zmiennej w nowej linii(załóżmy) może doprowadzić do zbędnego poszerzenia kodu?